customtransforms/custom_single_json_retriever.py

```diff
@@ -0,0 +1,59 @@
+from transformer import Transformer
+from utilities import flat
+
+
+class SingleJsonRetriever(Transformer):
+    '''
+    Transforms perfherder data into the standardized data format.
+    '''
+    entry_number = 0
+
+    def transform(self, data):
+        ret = []
+        self.entry_number += 1
+
+        for bs in data[0]['browserScripts']:
+            ret.append({
+                'data': bs['pageinfo']['navigationStartTime'],
+                'xaxis': self.entry_number,
+                'subtest': 'browserScripts.pageinfo.navigationStartTime'
+            })
+
+            flatted_timings = flat(bs['timings'], 'browserScripts.timings')
+
+            for k, v in flatted_timings.items():
+                ret.append({
+                    'data': v,
+                    'xaxis': self.entry_number,
+                    'subtest': k
+                })
+
+        return ret
+
+    def merge(self, data):
```

Comment:
This looks like a copy-paste from the `Testing` transformer? If so, maybe it would make more sense to add another merge function called `merge_subtests` in `Transformer`. Then, you could just call that function from here instead of duplicating code.

Reply:
Yes , I copy-paste  `merge()` function to see how my `transform()` works. This is a temporary solution. I may write a new `merge()` later.